extractors/hdf5imagingextractor/hdf5imagingextractor.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+from pathlib import Path
+from ..imagingextractor import ImagingExtractor
+
+try:
+    import h5py
+    HAVE_H5 =True
+except ImportError:
+    HAVE_H5 = False
+
+
+class Hdf5ImagingExtractor(ImagingExtractor):
+    extractor_name = 'Hdf5Imaging'
+    installed = HAVE_H5  # check at class level if installed or not
+    is_writable = True
+    mode = 'file'
+    installation_mesg = "To use the Hdf5 Extractor run:\n\n pip install h5py\n\n"  # error message when not installed
+
+    def __init__(self, filepath, sampling_frequency=None,
+                 channel_names=None):
+        assert HAVE_H5, self.installation_mesg
+        ImagingExtractor.__init__(self)
+        self.filepath = Path(filepath)
+        self._sampling_frequency = sampling_frequency
+        assert self.filepath.suffix in ['.tiff', '.tif']
+        # TODO placeholder
+        self._video = np.load(self.filepath)
+        self._channel_names = channel_names
+
+        if len(self._video.shape) == 3:
+            # 1 channel
+            self._num_channels = 1
+            self._num_frames, self._size_x, self._size_y = self._video.shape
```

Comment:
This is the order of dimensions that NWB wants, but it is often not the shape that is given. Maybe we could allow an optional arg for specifying the order of the dimensions